html-boilerplate/css/index.css

```
@@ -1,26 +1,88 @@
 
-body {background-color: powderblue;
-      font-family: verdana;
+body {
+  background-color: rgb(0, 0, 228);
+  font-family: verdana; 
+  display: flex;
+  flex-direction: column;
+  margin: 24px;
 }	
-p {font-family: "Sofia", sans-serif; font-size: 20px; margin: 5px; padding: 5px;}
-h1 {color: red; font-size: 30px;}
-h2 {color: royalblue; font-size: 30px;}
-ul {color: purple;}
+
+p {font-family: "Sofia", sans-serif; 
+  font-size: 20px;
+  margin: 5px; 
+  padding: 5px;}
+
+h1 {color: peachpuff; font-size: 30px;}
+h2 {color: peachpuff; font-size: 30px;}
+ul {color: black;}
```

We can still see the bullet points from the list, which looks out of place now that we have this in a grid. Let's either use `list-style: none` here to hide the bullets or replace the `ul` and `li` elements entirely and just use `div`. 
```suggestion
ul {
  color: black;
  list-style: none;
}
```